security_groups.tf

```diff
@@ -0,0 +1,45 @@
+resource "aws_security_group" "amongus_instance" {
+  name        = "amongus_instance"
+  description = "Security Group for the amongus instance"
+  vpc_id      = var.vpc_id
+}
+
+resource "aws_security_group_rule" "amongus_instance_http" {
+  type              = "ingress"
+  from_port         = 22023
+  to_port           = 22023
+  protocol          = "tcp"
```

Comment:
Out of the box, the base game (no plugins) runs on UDP/22023, so I think the two SG rules for 22022 and 22023 need to have their protocols switched:
22023 - UDP
22022 - TCP

I guess it could be deployed either way, but I think this is the defaults.